### PR TITLE
Add rs-consul client to community-provided SDKs list

### PR DESCRIPTION
**Pull request by JanBerktold**
We (Roblox) just published https://github.com/Roblox/rs-consul -- adding it to the list here for discovery by others.

### CODE REVIEW

File: website/content/api-docs/libraries-and-sdks.mdx [hunk 1]
```diff
@@ -131,4 +131,8 @@ the community.
     <a href="https://github.com/alphaHeavy/consul-haskell">consul-haskell</a> -
     Haskell client for the Consul HTTP API
   </li>
+  <li>
+    <a href="https://github.com/Roblox/rs-consul">rs-consul</a> -
+    Rust client Consul HTTP API
```
Comment (mikemorris) [inline website/content/api-docs/libraries-and-sdks.mdx:136]:
```suggestion
    Rust client for the Consul HTTP API
```

**Review by mikemorris — APPROVED** [review]
Thanks for adding this!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/400895.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e91996f1309573bfc51c4123bd433d5ee06ea116 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e91996f1309573bfc51c4123bd433d5ee06ea116 onto `release/1.10.x` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/401403.

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit e91996f1309573bfc51c4123bd433d5ee06ea116 onto `stable-website` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/401403)

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit e91996f1309573bfc51c4123bd433d5ee06ea116 onto `release/1.10.x` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/401403)

**Pull request merged**